src/icons/service-serverless-aws.svg

```diff
@@ -0,0 +1 @@
-.987-.378s-.365-.588-.365-1.009c0-.447.148-.809.449-1.082.3-.273.7-.41 1.209-.41.168 0 .34.016.523.042.182.026.37.068.567.115v-.383c0-.4-.079-.678-.232-.84-.158-.164-.424-.242-.804-.242-.173 0-.35.02-.533.068-.182.047-.36.105-.533.178a1.378 1.378 0 0 1-.173.069.286.286 0 0 1-.079.015c-.069 0-.103-.052-.103-.162v-.258c0-.084.01-.147.034-.184a.372.372 0 0 1 .139-.11c.172-.095.38-.174.621-.236.242-.069.499-.1.77-.1.587 0 1.017.142 1.293.425.271.284.41.715.41 1.293v1.702h.01zm-2.003.799c.163 0 .33-.031.508-.095.178-.063.336-.178.469-.336.079-.1.138-.21.168-.336.03-.126.049-.279.049-.457v-.221a3.887 3.887 0 0 0-.454-.09 3.498 3.498 0 0 0-.464-.03c-.33 0-.572.067-.735.21-.163.141-.242.34-.242.604 0 .247.06.43.183.557.118.13.291.194.518.194zm3.962.568c-.089 0-.148-.016-.187-.053-.04-.032-.074-.105-.104-.205l-1.16-4.062a.972.972 0 0 1-.044-.21c0-.084.04-.131.118-.131h.484c.094 0 .158.016.192.052.04.032.07.105.1.205l.828 3.479.77-3.479c.024-.105.054-.173.094-.205a.323.323 0 0 1 .197-.052h.395c.093 0 .158.016.197.052.04.032.074.105.094.205l.78 3.52.853-3.52a.46.46 0 0 1 .099-.205.307.307 0 0 1 .192-.052h.459c.079 0 .123.042.123.131 0 .026-.005.053-.01.084a.786.786 0 0 1-.034.132l-1.19 4.061c-.029.105-.063.174-.103.205a.302.302 0 0 1-.187.053h-.425c-.093 0-.157-.016-.197-.053-.04-.037-.074-.105-.094-.21l-.765-3.39-.76 3.385c-.024.105-.054.173-.093.21-.04.037-.109.053-.198.053h-.424zm6.34.141c-.256 0-.513-.031-.76-.094-.246-.063-.438-.132-.567-.21a.366.366 0 0 1-.153-.148.393.393 0 0 1-.03-.147v-.268c0-.11.04-.163.114-.163.03 0 .06.006.089.016.03.01.074.032.123.053.168.079.35.142.543.184.198.042.39.063.587.063.311 0 .553-.058.72-.174a.575.575 0 0 0 .257-.499.53.53 0 0 0-.133-.368c-.089-.1-.256-.189-.498-.273l-.716-.236c-.36-.121-.626-.3-.79-.536a1.305 1.305 0 0 1-.246-.762c0-.22.044-.415.133-.584.09-.168.207-.315.355-.43.149-.121.316-.21.514-.274a2.03 2.03 0 0 1 .621-.089c.109 0 .222.005.331.021.113.016.217.037.32.058.1.026.193.053.282.084.089.032.158.063.207.095.07.042.119.084.148.13.03.043.045.1.045.174v.247c0 .11-.04.169-.114.169a.49.49 0 0 1-.187-.064 2.14 2.14 0 0 0-.948-.204c-.281 0-.503.047-.656.147-.153.1-.232.252-.232.467 0 .147.05.273.148.373.099.1.281.2.543.29l.7.236c.356.12.612.289.765.504.153.216.227.463.227.736 0 .226-.044.43-.128.61a1.4 1.4 0 0 1-.36.462 1.558 1.558 0 0 1-.548.294c-.222.074-.454.11-.705.11z" fill="#252F3E"/><path fill-rule="evenodd" clip-rule="evenodd" d="M24.495 26.884c-1.623 1.276-3.982 1.954-6.01 1.954-2.842 0-5.402-1.119-7.337-2.979-.153-.147-.015-.347.168-.231a14.109 14.109 0 0 0 7.342 2.075c1.801 0 3.78-.399 5.6-1.219.272-.131.504.19.237.4z" fill="#F90"/><path fill-rule="evenodd" clip-rule="evenodd" d="M25.171 26.064c-.207-.284-1.372-.137-1.9-.068-.157.02-.182-.126-.039-.237.928-.693 2.452-.494 2.63-.262.178.236-.05 1.86-.918 2.637-.133.121-.261.058-.202-.1.197-.52.636-1.691.43-1.97z" fill="#F90"/></g><defs><clipPath id="a"><path fill="#fff" transform="translate(11 19.334)" d="M0 0h15v9.563H0z"/></clipPath></defs></svg>
```

Comment:
These are general change we make with svgs.
remove fill="none" in svg tag
change fill="currentColor" in path tag
Please make the change unless otherwise required

Reply:
There are multiple colors in this one, so can't do it.